File: code/go/ch11/src/luago/api/lua_vm.go
package api

type LuaVM interface {
	LuaState
	PC() int
	AddPC(n int)
	Fetch() uint32
	GetConst(idx int)
	GetRK(rk int)
	RegisterCount() int
	LoadProto(idx int)
	LoadVararg(n int)
}

File: code/csharp/ch09/luavm/api/LuaVM.cs
using luavm.binchunk;

namespace luavm.api
{
    public partial interface LuaState
    {
        int PC();
        void AddPC(int n);
        uint Fetch();
        void GetConst(int idx);
        void GetRK(int rk);
        int RegisterCount();
        void LoadVararg(int n);
        void LoadProto(int idx);
    }
}